nlp_models/tests/test_nlp_utils.py

```diff
@@ -0,0 +1,20 @@
+import unittest
```

Comment:
Need more working tests, not comments, create only few simple tests for now and check with db